modules/chapter2/pages/section1.adoc

```diff
@@ -1,3 +1,43 @@
-= Section 1
+= RHODS
 
-This is _Section 1_ of _Chapter 2_ in the *hello* quick course....
\ No newline at end of file
+This section covers how to set up a Data Science project in RHODS and how to create a new Workbench for us to work on our code
```

Comment:
This should be covered in Course 1 about RHODS intro. Anyway, no harm repeating it - maybe just do a quick double check of course 1 (https://redhatquickcourses.github.io/rhods-intro/) and talk to Jaime privately and see if there is too much duplication?

Reply:
Removed the pod error part and edited the creation of the project and the workbench. Now it just serves as a reference